src/werkzeug/wrappers/base_response.py

```diff
@@ -490,16 +490,34 @@ def set_cookie(
             ),
         )
 
-    def delete_cookie(self, key, path="/", domain=None):
+    def delete_cookie(
+        self, key, path="/", domain=None, secure=False, httponly=False, samesite=None,
```

Comment:
Not correctly formatted, but just a suggestion.
```suggestion
        self, key, path="/", domain=None, secure: bool=False, httponly: bool=False, samesite: str=None,
```

Reply:
Done, thanks.